src/server/pps/cmds/cmds.go

```diff
@@ -235,6 +237,8 @@ $ pachctl list-job -p foo bar/YYY
 		}),
 	}
 	rawFlag(listDatum)
+	listDatum.Flags().BoolVar(&paginate, "paginate", false, "Get a subset of the results")
+	listDatum.Flags().Int64Var(&page, "page", 0, "Specify the page of results to send")
```

Comment:
That `0` there is a default value so you can just change that to negative `-1` and you'll be able to see if it was specified or not. Also I don't really think unsigned ints are worth it, even though the value shouldn't ever be negative, you just wind up having to do a million casts.